python/thrift/appengine-ssl-gateway/README.md

```diff
@@ -0,0 +1,253 @@
+# Connecting from Google App Engine to Google Cloud Bigtable
+
+It's easy to deploy a Thrift gateway cluster on Google Compute Engine that
+proxies request to the Cloud Bigtable API.
+
+Here are step by step instructions for starting a Cloud Bigtable cluster and
+building and deploying a Thrift server on Compute Engine.
+
+**Table of contents**
```

Comment:
I just checked w/ our Python team, we don't deliver any samples w make.  If that's my only major comment however, I'll accept it, 'til we can do something more appropriate.